content/en/blog/_posts/2022-09-12-Announcing-the-Auto-Refreshing-Official-CVE-Feed/index.md

```diff
@@ -48,6 +48,14 @@ official Kubernetes CVEs
   releases through their work in Kubernetes Community - via various Special
   Interest Groups and Committees.
 
+## Implementation Details
+
+A supporting
+[contributor blog](https://kubernetes.dev/blog/2022/09/12/k8s-cve-feed-alpha/)
+was published that describes in depth on how this CVE feed was implemented to
+ensure the feed was reasonably protected against tampering and was automatically
```

Comment:
```suggestion
ensure the feed is reasonably protected against tampering and is automatically
```

Reply:
I agree with that, I wanted to propose that but the intent of the PR was just to reinstate a paragraph written and reviewed previously.